app/wama.core/ViewModel/User/UserAccountViewModel.cs

```diff
@@ -6,7 +6,7 @@ namespace WAMA.Core.ViewModel.User
     public abstract class UserAccountViewModel
     {
         public virtual UserAccountType AccountType { get; }
-
+        [Required]
```

Comment:
You've got to provide detailed information on how to get this buy you're talking about.